### ISSUE TITLE
Questions about testing

### ISSUE THREAD

**Issue by fyf12138** [SeungjunNah/DeepDeblur-PyTorch#52]
Pls where can I find the image after deblur in the test set?

**Comment by fyf12138** [comment]
> Hello! I would like to ask if I stopped training it when I trained to 150 epochs on the GOPRO_Large dataset. But now I want to train from the 150 round and train to the 200 epochs, how do I enter the commands?

**Comment by SeungjunNah** [comment]
Check out `--save_dir` argument.

https://github.com/SeungjunNah/DeepDeblur-PyTorch/blob/e92dc7c3337127887da617ffe819b49cb022fd81/src/option.py#L109
